com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/ReadOnlyParametersChecker.java

```diff
@@ -0,0 +1,62 @@
+package com.sap.cloud.lm.sl.cf.process.util;
+
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.READ_ONLY_MODULE_PARAMETERS;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.READ_ONLY_RESOURCE_PARAMETERS;
+import static com.sap.cloud.lm.sl.cf.core.model.SupportedParameters.READ_ONLY_SYSTEM_PARAMETERS;
+
+import java.text.MessageFormat;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javax.inject.Named;
+
+import com.google.common.collect.Sets;
+import com.sap.cloud.lm.sl.cf.process.message.Messages;
+import com.sap.cloud.lm.sl.common.SLException;
+import com.sap.cloud.lm.sl.mta.model.DeploymentDescriptor;
+import com.sap.cloud.lm.sl.mta.model.NamedElementWithParametersContainer;
+
+@Named
+public class ReadOnlyParametersChecker {
+
+    public void check(DeploymentDescriptor descriptor) {
+        Map<String, Set<String>> commonReadOnlyParameters = new LinkedHashMap<>();
```

Comment:
Rename the variable because it is not common any more

Reply:
Done